ktlint-rule-engine/src/test/kotlin/com/pinterest/ktlint/rule/engine/internal/SuppressionLocatorBuilderTest.kt

```diff
@@ -355,14 +355,15 @@ class SuppressionLocatorBuilderTest {
 
         val actual =
             KtLintRuleEngine(
-                ruleProviders = setOf(
-                    RuleProvider { IndentationRule() },
-                ),
-                editorConfigOverride =
-                EMPTY_EDITOR_CONFIG_OVERRIDE
-                    .plus(
-                        STANDARD_NO_FOO_IDENTIFIER_RULE_ID.createRuleExecutionEditorConfigProperty() to RuleExecution.enabled,
+                ruleProviders =
```

Comment:
@paul-dingemans under which rule this got formatted?

Reply:
I am not sure whether I understand.

But the IndentationRule would format:
```
                return mapOf(
                       1 to "   1 ms",
                      10 to "  10 ms",
                     999 to " 999 ms",
                    1000 to "   1 sec",
                )
```
to
```
                return mapOf(
                    1 to "   1 ms",
                    10 to "  10 ms",
                    999 to " 999 ms",
                    1000 to "   1 sec",
                )
```
if the rule would not be suppressed.

Does that answer the question?

Reply:
Actually, I am wondering why can't `ruleProviders = setOf(` be on the same line? Is this coz ktlint official recommends to always(in case of multiline chain) to newline after `=`